update rke2 node drain default value

mantis-toboggan-md opened the pull request:
<!-- This template is for Devs to give QA details before moving the issue To-Test -->
### Summary
Fixes #9108 

Per internal ref, this requested change is for provisioned clusters - see comment on original issue re: imported cluster UI

### Occurred changes and/or fixed issues
* updated the rke2/k3s upgrade strategy configuration to default to drain nodes 'yes'
* updated DrainOptions component to use i18n
* added unit tests for DrainOptions
* added v-clean-tooltip support to jest config (I was getting `Failed to resolve directive: v-clean-tooltip`)

cnotv commented:
@mantis-toboggan-md looks good the setup for Jest. Did exactly the same here 
 :D https://github.com/rancher/dashboard/pull/9264

cnotv commented on shell/edit/provisioning.cattle.io.cluster/DrainOptions.vue:
```diff
@@ -51,6 +52,8 @@ export default {
     this.update();
   },
 
+  computed: { ...mapGetters({ t: 'i18n/t' }) },
```
Nvm, we are not using the CRUResource mixin here.

cnotv reviewed (approved):
LGTM

mantis-toboggan-md commented:
The internal ticket was updated with more info: the actual field to change is the 'drain' options in the machine pool advanced section. I've updated this PR to set that true by default, added a unit test, and reverted the upgrade strategy defaults change. 

![Screen Shot 2023-09-27 at 2 58 04 PM](https://github.com/rancher/dashboard/assets/42977925/9cf37454-d83a-42a7-a932-139f3aa68064)

momesgin reviewed (approved):
LGTM
